pcdsdevices/areadetector/detectors.py

```diff
@@ -209,6 +215,21 @@ class PCDSAreaDetectorTyphos(Device):
                     num_dimensions='ndimensions',
                     kind='normal')
 
+    # Eventually I want to typhos-ify into a button. I couldn't figure out
+    # how to do that, so I will settle for this for now. 
+    def viewer(self):
+        """
+        Launch the python camera viewer for this camera.
+        """
+        arglist = ['/reg/g/pcds/pyps/apps/camviewer/latest/run_viewer.sh',
+                   '--instrument',
+                   '{0}'.format(get_hutch().lower()),
+                   '--oneline',
+                   'GE:16,{0}:IMAGE1;{0},,{0}'.format(self.prefix[0:-1])]
```

Comment:
Thoughts on `IMAGE1` vs `IMAGE2`? I know that sometimes `IMAGE1` is the data stream, sometimes it's a viewer.

Reply:
Yeah, I've seen both as well. My reason for choosing IMAGE1 is that some stripped down areaDetector IOCs only have IMAGE1, the Thorlabs WFS IOC, a candidate for this class, being the one I can think of off the top of my head. The VISAR IOC may be this way as well. This may not be the best reason, but it's all I have. :)

We could make this method a little smarter and have it figure out which stream to use if we find that we have a mix of cameras using IMAGE1 vs IMAGE2. It would be nice if we could standardize the purpose of the image streams internally to avoid this kind of thing.

Reply:
That's pretty good reasoning. Let's stick with IMAGE1 and maybe revisit it later.